server/order/order.go

```diff
@@ -260,7 +260,7 @@ func (p *Prefix) Base() uint32 {
 	return p.BaseAsset
 }
 
-// Base returns the  quote asset integer ID.
+// Quote returns the  quote asset integer ID.
```

Comment:
Would you please remove the extra whitespace while you're here?  Thanks for the edit.

Reply:
it's ok.